XenAdmin/SettingsPanels/PoolGpuEditPage.resx

```diff
@@ -214,7 +214,7 @@
     <value>3</value>
   </data>
   <data name="radioButtonDepth.Text" xml:space="preserve">
-    <value>Maximum density: put as many VMs as possible on the same GPU</value>
+    <value>&amp;Maximum density: put as many VMs as possible on the same GPU</value>

```

Comment:
Would it be possible to have 'd' for 'Maximum **d**ensity' and 'p' for Maximum **p**erformance?
And 'n' for 'This server will **n**ot use...'